README.md

```diff
@@ -31,89 +42,95 @@ DATABASE_PASSWORD=___
 DATABASE_NAME=___
 DATABASE_HOST_ADDRESS=___
 MAPBOX_API_KEY=___
+LOG_CONFIG_PATH=___
+LOG_FILE_PATH=___
+PYTHONPATH=___
+SLACKBOT_TOKEN=___
 ```
 
-Ask someone on the Data team for the actual environment variables you'll need! 
+Ask someone on the Data team for the actual environment variables you'll need!
 
 ## Postgres
 
 1. Download and install Postgres.
 
-    From the source: [https://www.postgresql.org/download/](https://www.postgresql.org/download/)
+   From the source: [https://www.postgresql.org/download/](https://www.postgresql.org/download/)
 
-    Or, on a Mac: `brew install postgresql`
+   Or, on a Mac: `brew install postgresql`
 
-2. Download and install pgAdmin 4, a management tool for Postgres. 
+2. Download and install pgAdmin 4, a management tool for Postgres.
```

Comment:
We were also able to brew install pgadmin4 right? We should add that as an option here

Reply:
Great catch, will add